README.adoc

```diff
@@ -0,0 +1,57 @@
+= IoT OpenShift Demo
+
+:Author:    Andrew Block, Ishu Verma
+:Email:     ablock@redhat.com, iverma@redhat.com
+:Date:      10/10/2016
+
+== Overview
+Demonstration of Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Gateway enables IoT data transformation, routing and real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Each component of the demo runs in its own container, thereby, allowing security, scalability and ease of deployment.
+
+image:images/ose-iot.png[width="500", height="400", align="center"]
```

Comment:
yes, how does this one look
![iot-ose-picture](https://cloud.githubusercontent.com/assets/16406768/19668931/1eb2c82e-9a10-11e6-9d50-f515bfcc8086.png)